source/features/sort-conversations-by-update-time.tsx

```diff
@@ -1,24 +1,23 @@
 import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
 
 import features from '.';
 import SearchQuery from '../github-helpers/search-query';
 
 function init(): void {
 	// Get issues links that don't already have a specific sorting applied
-	for (const link of select.all(`
-		a[href*="/issues"]:not([href*="sort%3A"]):not(.issues-reset-query),
-		a[href*="/pulls" ]:not([href*="sort%3A"]):not(.issues-reset-query)
-	`)) {
+	const issueLinks = select.all<HTMLAnchorElement>(':is(a[href*="/issues"], a[href*="/pulls"]):not([href*="sort%3A"]):not(.issues-reset-query)');
+	for (const link of issueLinks) {
 		// Pick only links to lists, not single issues
 		// + skip pagination links
 		// + skip pr/issue filter dropdowns (some are lazyloaded)
-		if (/(issues|pulls)\/?$/.test(link.pathname) && !link.closest('.pagination, .table-list-filters')) {
+		if (pageDetect.isConversationList(link) && !link.closest('.pagination, .table-list-filters')) {
```

Comment:
Does this solve https://github.com/sindresorhus/refined-github/issues/4142? It does make the test more specific, but it doesn’t check the host. Maybe it needs an extra `link.host === location.host`

Reply:
I does fix it. I don't think the host check is needed. Since that would mean GHE adding a link to a issue list on github would not be sorted

Reply:
It needs it. It only works in this specific case because the underlying isRepo excludes as few keywords from being recognized as repositories, like “docs”. Change that keyword and you’ll see that this issue isn’t fixed
![E4001A1C-CD93-4C16-B7E9-D19A102EAF60](https://user-images.githubusercontent.com/1402241/115331635-772a8480-a1c0-11eb-9362-639195b4f3b2.jpeg)